micromouse/AssertionHandler.h

```diff
@@ -0,0 +1,45 @@
+//
+//  AssertionHandler.h
+//  micromouse-xcode
+//
+//  Created by Erick Sanchez on 4/20/16.
+//  Copyright © 2016 Erick Sanchez. All rights reserved.
+//
+
+#ifndef AssertionHandler_h
+#define AssertionHandler_h
+
+#include "Memory.h"
+#include <assert.h>
+
+#ifdef __MK20DX256__ // Teensy Compile
+
+inline static void logError( int code ) {
+    //Read and increment to the next error state, as well as validate to loop back to 1 if incremented to 21
+    int intNextIndex = Micromouse::Memory::read(Micromouse::ERROR_MEMORY) +1;
+    intNextIndex = intNextIndex > 20 ? 1 : intNextIndex;
+    
+    //Find next state address and save code as well as the most recent error state in ERROR_MEMORY
+    int intNextStateAddress = Micromouse::ERROR_MEMORY +intNextIndex * 4;
+    Micromouse::Memory::write( intNextStateAddress, code);
+    Micromouse::Memory::write( Micromouse::ERROR_MEMORY, intNextIndex);
+    
+}
+
```

Comment:
how about now???
plus i can't tell why i can't call to this `inline` method anywhere in the project :P